### ISSUE TITLE
Publish artifacts for v2.7.1

### ISSUE THREAD

**Issue by IzeTPL** [hagay3/skuber#50]
Hi,
Since you've created tag for `v2.7.1` could you publish artifacts for it?
Thanks

**Comment by hagay3** [comment]
@IzeTPL 
There was a degradation in the build, I cant build sbt on my machine with the latest changes.
I'm still investigating
It's the same error for all API endpoints
in order to reproduce:
sbt
skuber/assembly
````
[error] /Users/hagai.ovadia/Hagai/mydev/skuber/client/src/main/scala/skuber/api/client/KubernetesClient.scala:296:76: type arguments [O] do not conform to trait ResourceDefinition's type parameter bounds [T <: skuber.TypeMeta]
[error]   def scale[O <: ObjectResource](objName: String, count: Int)(implicit rd: ResourceDefinition[O], sc: Scale.SubresourceSpec[O], lc: LoggingContext): Future[Scale]
[error]                                                                            ^
[error] /Users/hagai.ovadia/Hagai/mydev/skuber/client/src/main/scala/skuber/api/client/KubernetesClient.scala:309:58: type arguments [O] do not conform to trait ResourceDefinition's type parameter bounds [T <: skuber.TypeMeta]
[error]       (implicit patchfmt: Writes[P], fmt: Format[O], rd: ResourceDefinition[O], lc: LoggingContext = RequestLoggingContext()): Future[O]
[error]                                                          ^
[error] /Users/hagai.ovadia/Hagai/mydev/skuber/client/src/main/scala/skuber/api/client/KubernetesClient.scala:319:73: type arguments [O] do not conform to trait ResourceDefinition's type parameter bounds [T <: skuber.TypeMeta]
[error]   def jsonMergePatch[O <: TypeMeta](obj: O, patch: String)(implicit rd: ResourceDefinition[O], fmt: Format[O], lc: LoggingContext): Future[O]
[error]        
````

**Comment by IzeTPL** [comment]
Works fine for me right now. I've had similar issues when I was testing local changes from PR and `sbt clean cleanFiles` solved problems. It seems that some cached files are causing this issues.

If you have any other issues I'm happy to help :)

**Comment by hagay3** [comment]
Thanks @IzeTPL 
Now I see another issue
```
[error] 1 error was encountered during merge
[error] java.lang.RuntimeException: deduplicate: different file contents found in the following:
[error] /home/travis/.cache/coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.10.5/jackson-annotations-2.10.5.jar:module-info.class
[error] /home/travis/.cache/coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-core/2.10.5/jackson-core-2.10.5.jar:module-info.class
[error] /home/travis/.cache/coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.10.5.1/jackson-databind-2.10.5.1.jar:module-info.class
[error] /home/travis/.cache/coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/datatype/jackson-datatype-jdk8/2.10.5/jackson-datatype-jdk8-2.10.5.jar:module-info.class
[error] /home/travis/.cache/coursier/v1/https/repo1.maven.org/maven2/com/fasterxml/jackson/datatype/jackson-datatype-jsr310/2.10.5/jackson-datatype-jsr310-2.10.5.jar:module-info.class
[error] 	at sbtassembly.Assembly$.applyStrategies(Assembly.scala:203)
[error] 	at sbtassembly.Assembly$.x$1$lzycompute$1(Assembly.scala:47)
[error] 	at sbtassembly.Assembly$.x$1$1(Assembly.scala:45)
[error] 	at sbtassembly.Assembly$.stratMapping$lzycompute$1(Assembly.scala:45)
[error] 	at sbtassembly.Assembly$.stratMapping$1(Assembly.scala:45)
```
I tried to fix it in this PR #51 
you can see the errors in travis build
https://travis-ci.com/github/hagay3/skuber/jobs/512520274

**Comment by hagay3** [comment]
@IzeTPL 
I fixed the issue in the following PR #51 
You should see the release in the coming 24 hours in maven repo.
Let me know if I can close this issue.
